### PR TITLE
[181] new service details view 

### PR DESCRIPTION
**Pull request by kmarszalek**

### CODE REVIEW

**Comment by mkasztelnik** [comment]
Please use EN language :smile:

File: spec/factories/services.rb [hunk 1]
```diff
@@ -13,5 +13,18 @@
       sequence(:connected_url) { "https://sample.url" }
       sequence(:open_access) { true }
     end
+    sequence(:webpage_url) { "https://sample.url"  }
+    sequence(:manual_url) { "https://sample.url"  }
+    sequence(:helpdesk_url) { "https://sample.url"  }
+    sequence(:tutorial_url) { "https://sample.url"  }
+    sequence(:terms_of_use_url) { "https://sample.url"  }
+    sequence(:corporate_sla_url) { "https://sample.url"  }
+    sequence(:access_policies_url) { "https://sample.url"  }
```
Comment (mkasztelnik) [inline spec/factories/services.rb]: Please use real sequence here and dedicated urls (e.g. https://acesspolicies#{i}.pl)

File: app/models/service.rb [hunk 2]
```diff
@@ -22,6 +22,18 @@ class Service < ApplicationRecord
   validates :connected_url, presence: true, url: true, if: :open_access?
   validates :provider, presence: true
   validates :rating, presence: true
+  validates :places, presence: true
+  validates :languages, presence: true
+  validates :dedicated_for, presence: true
+  validates :terms_of_use_url, presence: true, url: true
```
[the remark this answers is not in the document]
Comment (kmarszalek) [reply app/models/service.rb]: done

File: spec/factories/services.rb (hunk 1 above)
Comment (kmarszalek) [reply spec/factories/services.rb]: done

**Pull request merged**